html/Axit/index.html

```diff
@@ -0,0 +1,591 @@
+    </div>
+
+    <div class="standard-picture__city">
+      <picture>
+        <source media="min-width:1024px" srcset="./assets/standard-picture/image-right-standard-u-262@3x.png">
+        <img src="./assets/standard-picture/image-right-standard-u-262@2x.png" alt="Some city view">
+      </picture>
+    </div>
+
+  </section>
+
+  <!-- Awsome things -->
+  <section class="awsome-things">
+    <h2 class="title title--big awsome-things__title">
+      Why this is awsome
+    </h2>
+
+    <span class="separator awsome-things__separator"></span>
+
+    <p class="description awsome-things__description">
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+    </p>
+
+    <div class="awsome-things__services">
+      <!-- Service 1 -->
+      <div class="awsome-things__service awsome-things-service">
+        
+        <img src="./assets/awsome-things/lightbulb.png" alt="" class="awsome-things-service__icon">
+
+        <p class="title title--bold awsome-things-service__title">
+          Thoughtful Design
+        </p>
+
+        <p class="description awsome-things-service__description">
+          Lorem ipsum dolor sit amet, consectetur adipiscing 
+          elit. Aenean euismod bibendum laoreet. Proin gravida 
+          dolor sit amet lacus accumsan et viverra.
+        </p>
+        
+      </div>
+
+      <!-- Service 2 -->
+      <div class="awsome-things__service awsome-things-service">
+        
+        <img src="./assets/awsome-things/happy.png" alt="" class="awsome-things-service__icon">
+
+        <p class="title title--bold awsome-things-service__title">
+          Well Crafted
+        </p>
+
+        <p class="description awsome-things-service__description">
+          Lorem ipsum dolor sit amet, consectetur adipiscing 
+          elit. Aenean euismod bibendum laoreet. Proin gravida 
+          dolor sit amet lacus accumsan et viverra.
+        </p>
+        
+      </div>
+
+      <!-- Service 3 -->
+      <div class="awsome-things__service awsome-things-service">
+        
+        <img src="./assets/awsome-things/flash.png" alt="" class="awsome-things-service__icon">
+
+        <p class="title title--bold awsome-things-service__title">
+          Easy to Customize
+        </p>
+
+        <p class="description awsome-things-service__description">
+          Lorem ipsum dolor sit amet, consectetur adipiscing 
+          elit. Aenean euismod bibendum laoreet. Proin gravida 
+          dolor sit amet lacus accumsan et viverra.
+        </p>
+        
+      </div>
+    </div>
+    
+  </section>
+ 
+  <!-- Pricing -->
+  <section class="pricing">
+      <h2 class="title title--big pricing__title">Pricing option</h2>
+
+      <span class="separator pricing__separator "></span>
+
+      <p class="description pricing__description">
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+      </p>
+
+      <div class="pricing__prices">
+
+        <!-- Price 1 -->
+        <button class="price pricing__price">
```

Comment:
I'm not sure if it button, as probably it is not and it is just a block, generally your button will have no other tags, maybe some spans